core/src/test/java/kafka/test/MockController.java

```diff
@@ -250,20 +238,18 @@ private ApiError incrementalAlterResource(ConfigResource resource,
     }
 
     @Override
-    public CompletableFuture<AlterPartitionReassignmentsResponseData>
-            alterPartitionReassignments(AlterPartitionReassignmentsRequestData request) {
+    public CompletableFuture<AlterPartitionReassignmentsResponseData> alterPartitionReassignments(AlterPartitionReassignmentsRequestData request) {
         throw new UnsupportedOperationException();
     }
 
     @Override
-    public CompletableFuture<ListPartitionReassignmentsResponseData>
-            listPartitionReassignments(ListPartitionReassignmentsRequestData request) {
+    public CompletableFuture<ListPartitionReassignmentsResponseData> listPartitionReassignments(ListPartitionReassignmentsRequestData request) {
         throw new UnsupportedOperationException();
     }
 
     @Override
     public CompletableFuture<Map<ConfigResource, ApiError>> legacyAlterConfigs(
-            Map<ConfigResource, Map<String, String>> newConfigs, boolean validateOnly) {
+                                                                               Map<ConfigResource, Map<String, String>> newConfigs, boolean validateOnly) {
```

Comment:
One question here. If you put the parameters of this method on the same line as the method name and you run again the formatter, will the parameters stay on the same line as the method name?
What would happen if you put the first parameter on the same line as the method name and then put each of the remaining parameters on its line own line?

Reply:
case 1. If you put the parameters of this method on the same line as the method name and run formatter: remains one line.
case 2. If you put the parameters separated and run formatter: broken. (below)
![20210606-101022](https://user-images.githubusercontent.com/2375128/120909374-72a62680-c6af-11eb-86e9-b09fa515c684.png)

Reply:
I do not know what you exactly mean with "broken", but that is the behavior I hoped for.

Reply:
Okay. I will fix it with the second approach.

Reply:
Could you put the first parameter in the same line as the method name and put the remaining parameters on their own line as I proposed? Could you do the same for `incrementalAlterConfigs()`?

If anybody does not like this style, we can still delete
```
<setting id="org.eclipse.jdt.core.formatter.alignment_for_parameters_in_constructor_declaration" value="18"/>
<setting id="org.eclipse.jdt.core.formatter.alignment_for_parameters_in_method_declaration" value="18"/>
```
from the formatter configuration, right?

Reply:
After that, I think, we can merge. Besides the import order changes, you kept the just a few formatting style changes, which I think is good. If somebody does not agree, we can make it even less invasive.